cloud-function/src/app.ts

```diff
@@ -39,6 +40,7 @@ router.all(
 router.all("/submit/mdn-yari/*", requireOrigin(Origin.main), proxyTelemetry);
 router.all("/pong/*", requireOrigin(Origin.main), express.json(), proxyPong);
 router.all("/pimg/*", requireOrigin(Origin.main), proxyPong);
+router.use(redirectNonCanonicals);
```

Comment:
Can we reorder the routes here? Move everything that doesn't need the `redirectNonCanonicals` up (I guess only get / and the assests).

Otherwise ready to go.

Reply:
@fiji-flo I moved `redirectNonCanonicals` down instead (and the `search-index.json` route up next to the assets route), to make the change less invasive: https://github.com/mdn/yari/pull/11151/files/55d08d37374c549d154529d8228dabcca4fc9a2d..45501138daec268e2fdb0dfa7a6ab9229f99f4b3